testing/trino-testing/src/main/java/io/trino/testing/AbstractTestJoinQueries.java

```diff
@@ -2285,6 +2286,7 @@ public void testMultiJoinWithEligibleForDynamicFiltering()
     }
 
     @Test
+    @Flaky(issue = "https://github.com/trinodb/trino/issues/5172", match = ".*expected.*but found.*")
```

Comment:
That's vague, but... yeah....

Reply:
yup. the numbers inside are kind of flaky too